antsibull/build_ansible_commands.py

```diff
@@ -164,28 +202,36 @@ def build_single_command():
         print(f'{app_ctx.extra["build_file"]} is for version {build_ansible_version} but we need'
               f' {app_ctx.extra["ansible_version"].major}'
               f'.{app_ctx.extra["ansible_version"].minor}')
-        return 1
+        return None
 
     with tempfile.TemporaryDirectory() as tmp_dir:
         download_dir = os.path.join(tmp_dir, 'collections')
         os.mkdir(download_dir, mode=0o700)
 
         # Download included collections
+        versions: t.Optional[t.Mapping[str, SemVer]] = None
```

Comment:
Could we do:
```suggestion
        versions: t.Dict[str, SemVer]] = {}
```
instead?  That way we're passing in the same type in both code paths.

Reply:
Fixed in b29791aa4070b0625f6845fe3382a7bca7dc17e0.